CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Fix nox's deprecated `session.install()` calls
 * Re-enable changelog validation in CI
 
+### Changed
```

Comment:
it's not a public interface the library users care about, therefore please put it in the `### Infrastructure` section

Reply:
Done.